ovh/resource_dbaas_logs_input_test.go

```diff
@@ -152,12 +153,17 @@ func testSweepDbaasInput(region string) error {
 
 func TestAccResourceDbaasLogsInput_basic(t *testing.T) {
 	serviceName := os.Getenv("OVH_DBAAS_LOGS_SERVICE_TEST")
+	name := "LOGSTASH"
+	version := "7.x"
```

Comment:
I think it should be useful to put in environment variable the version, if not at each version deprecation or removal, this test should be updated